setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
there's no requirements.in template file for it, have any example of how should I be supposed to use it? (btw. adding the packages passed fixed some of the issues in the tests...)